### PR TITLE
Fix upstream K8s bump

### PR DESCRIPTION
**Pull request by awgreene**

### CODE REVIEW

**Comment by awgreene** [comment]
/hold
This is a test PR, should not be merged.

**Comment by awgreene** [comment]
/unhold
Created #2621 to capture the issue with the versioned client, planning to have this merged.

File: pkg/api/client/clientset/versioned/typed/operators/v1/operators_client.go [hunk 1]
```diff
@@ -54,12 +56,28 @@ func (c *OperatorsV1Client) OperatorGroups(namespace string) OperatorGroupInterf
 }
 
 // NewForConfig creates a new OperatorsV1Client for the given config.
+// NewForConfig is equivalent to NewForConfigAndClient(c, httpClient),
+// where httpClient was generated with rest.HTTPClientFor(c).
 func NewForConfig(c *rest.Config) (*OperatorsV1Client, error) {
```
Comment (tylerslaton) [inline pkg/api/client/clientset/versioned/typed/operators/v1/operators_client.go:61]: Should we be keeping these NewForConfigs around? Are they still going to be used?

Comment (tylerslaton) [inline pkg/api/client/clientset/versioned/typed/operators/v1/operators_client.go:61]: non-blocking: Are changes like this generated or did you add manually add them?

Comment (timflannagan) [reply pkg/api/client/clientset/versioned/typed/operators/v1/operators_client.go:61]: Those are manually generated IIRC.

File: pkg/api/client/clientset/versioned/clientset.go [hunk 2]
```diff
@@ -116,14 +136,11 @@ func NewForConfig(c *rest.Config) (*Clientset, error) {
...
 func NewForConfigOrDie(c *rest.Config) *Clientset {
-	var cs Clientset
-	cs.operatorsV1alpha1 = operatorsv1alpha1.NewForConfigOrDie(c)
-	cs.operatorsV1alpha2 = operatorsv1alpha2.NewForConfigOrDie(c)
-	cs.operatorsV1 = operatorsv1.NewForConfigOrDie(c)
-	cs.operatorsV2 = operatorsv2.NewForConfigOrDie(c)
-
-	cs.DiscoveryClient = discovery.NewDiscoveryClientForConfigOrDie(c)
-	return &cs
+	cs, err := NewForConfig(c)
```
Comment (tylerslaton) [inline pkg/api/client/clientset/versioned/clientset.go:139]: Nice, I like this better, definitely cleaner.

**Pull request merged**